javamelody-core/src/main/java/net/bull/javamelody/Parameter.java

```diff
@@ -403,7 +403,17 @@
 	/**
 	 * <a href='https://www.datadog.com/'>Datadog</a> host for accessing API. 'api.datadoghq.com' is default.
 	 */
-	DATADOG_API_HOST("datadog-api-host");
+	DATADOG_API_HOST("datadog-api-host"),

```

Comment:
If this PR should be merged, I have a few remarks. I will probably have other remarks later.